rolldown.config.js

```diff
@@ -23,6 +23,7 @@ export default defineConfig({
     }
   ],
   platform: 'browser',
+  // @ts-expect-error TODO
```

Comment:
How to fix this?

Reply:
Looks like a [bug](https://github.com/rolldown/rolldown/issues/3407) with the rolldown types.